setup.py

```diff
@@ -18,18 +40,21 @@
     'License :: OSI Approved :: GNU General Public License (GPL)',
     'Natural Language :: English',
     'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 3',
     'Topic :: Software Development']
 
 setup(name='ropemode',
-      version=ropemode.VERSION,
-      description=ropemode.INFO,
+      version=VERSION,
+      description=INFO,
       author='Ali Gholami Rudi',
       author_email='aligrudi@users.sourceforge.net',
       url='http://rope.sf.net/',
       packages=['ropemode'],
       license='GNU GPL',
       classifiers=classifiers,
-      requires=['rope (>= 0.9.4)'],
+      requires=requires,
+      use_2to3=True,
```

Comment:
Do we use ``2to3``? I would prefer to make the code working both under py2k and py3k natively.